app/controllers/OverviewController.scala

```diff
@@ -298,16 +301,29 @@ class OverviewController @Inject()(
     identifyAndRequireData(srn, taxYear, version).async { implicit request =>
       reportType match {
         case PsrReportType.Sipp.name =>
-          val sippUrl = s"${config.urls.sippBaseUrl}/${srn.value}${config.urls.sippContinueJourney}"
-          Future.successful {
-            val result = Redirect(sippUrl)
-              .addingToSession(Constants.TAX_YEAR -> taxYear)
-              .addingToSession(Constants.VERSION -> version)
+          version.toIntOption match {
+            case None =>
+              logger.error(s"Could not parse version [$version] to int")
```

Comment:
Could you please add something to indicate this error message is for SIPP? This is so if we see a pager duty alert, to know that this relates to SIPP.